docs/tutorials/classification.ipynb

```diff
@@ -2,195 +2,433 @@
+       "| min       |  0.16670 |   0.00000 |   1.00000 | … |   0.00000 |   0.00000 |   0.00000 |   0.00000 |\n",
+       "| max       | 80.00000 |   8.00000 |   3.00000 | … |   1.00000 |   1.00000 |   1.00000 |   1.00000 |\n",
+       "| mean      | 29.88113 |   0.49885 |   2.29488 | … |   0.35600 |   0.69824 |   0.20626 |   0.09396 |\n",
+       "| median    | 29.88113 |   0.00000 |   3.00000 | … |   0.00000 |   1.00000 |   0.00000 |   0.00000 |\n",
+       "| standard  | 12.88320 |   1.04166 |   0.83784 | … |   0.47900 |   0.45920 |   0.40478 |   0.29189 |\n",
+       "| deviation |          |           |           |   |           |           |           |           |\n",
+       "| distinct  | 99.00000 |   7.00000 |   3.00000 | … |   2.00000 |   2.00000 |   2.00000 |   2.00000 |\n",
+       "| value     |          |           |           |   |           |           |           |           |\n",
+       "| count     |          |           |           |   |           |           |           |           |\n",
+       "| idness    |  0.07563 |   0.00535 |   0.00229 | … |   0.00153 |   0.00153 |   0.00153 |   0.00153 |\n",
+       "| missing   |  0.00000 |   0.00000 |   0.00000 | … |   0.00000 |   0.00000 |   0.00000 |   0.00000 |\n",
+       "| value     |          |           |           |   |           |           |           |           |\n",
+       "| ratio     |          |           |           |   |           |           |           |           |\n",
+       "| stability |  0.20092 |   0.68067 |   0.54163 | … |   0.64400 |   0.69824 |   0.79374 |   0.90604 |\n",
+       "+-----------+----------+-----------+-----------+---+-----------+-----------+-----------+-----------+"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
    ],
+   "source": [
+    "transformed_raw_data.summarize_statistics()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Spliting the `raw_data` into train and test sets\n",
+    "- **Training set**: Contains 60% of the data and will be used to train the model.\n",
+    "- **Testing set**: Contains 40% of the data and will be used to test the model's accuracy."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_table, test_table = transformed_raw_data.shuffle_rows().split_rows(0.6)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
    "metadata": {
     "collapsed": false
    },
-   "execution_count": null,
-   "outputs": []
+   "source": [
+    "### Mark the `survived` column as the target variable to be predicted"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "tagged_train_table = train_table.to_tabular_dataset(\"survived\")"
```

Comment:
Here you can declare the column `"name"` as an `extra_name`.

Reply:
but we are removing it from `raw_data` so its not in the `train_data` won't it be useless then ?

Reply:
Keep it in the `raw_data`, make it an extra here.